instance creation
new
	self error: 'Use #table:'